Launcher/Classes/ViewControllers/TasksViewController.swift

```diff
@@ -278,13 +249,23 @@ class TasksViewController: NSViewController {
         }
     }
     
-    func getSimulatorsCommand() {
-        if let launchPath = Constants.FilePaths.Bash.simulators {
+    func getDevicesCommand(ofType type: Constants.DeviceType) {
+        var path: String?
+        
+        switch type {
+        case .simulator:
+            path = Constants.FilePaths.Bash.simulators
+        case .physical:
+            path = Constants.FilePaths.Bash.physicalDevices
+        }
+        
+        if let launchPath = path {
             let outputStream = CommandTextOutputStream()
             outputStream.textHandler = { text in
-                let filderedText = text.components(separatedBy: "\n").filter { $0.contains("Simulator") }
+                let filteredText = text.components(separatedBy: "\n").filter { !RegexHandler().matches(for: "\\(([^()])*\\) \\[(.*?)\\]", in: $0).isEmpty }
```

Comment:
I would maybe do a `== false` in the `filter` instead of the `!` at the beginning. Especially since you have the `matches` (but are actually checking the `isEmpty`), this is pretty easily misread.